server/routes/education-support-plan/create/index.ts

```diff
@@ -0,0 +1,87 @@
+import { NextFunction, Request, Response, Router } from 'express'
+import { Services } from '../../../services'
+import insertJourneyIdentifier from '../../../middleware/insertJourneyIdentifier'
+import setupJourneyData from '../../../middleware/setupJourneyData'
+import asyncMiddleware from '../../../middleware/asyncMiddleware'
+
+const createEducationSupportPlanRoutes = (services: Services): Router => {
+  const { journeyDataService } = services
+  const router = Router({ mergeParams: true })
+
+  router.use('/', [
+    // TODO - enable this line when we understand the RBAC roles and permissions
+    // checkUserHasPermissionTo(ApplicationAction.RECORD_EDUCATION_LEARNER_SUPPORT_PLAN),
+    insertJourneyIdentifier({ insertIdAfterElement: 3 }), // insert journey ID immediately after '/education-support-plan/:prisonNumber/create' - eg: '/education-support-plan/A1234BC/create/473e9ee4-37d6-4afb-92a2-5729b10cc60f/who-created-the-plan'
+  ])
+  router.use('/:journeyId', [setupJourneyData(journeyDataService)])
+
+  router.get('/:journeyId/who-created-the-plan', [
+    asyncMiddleware(async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+      res.send('Who create the plan page')
```

Comment:
Hardcoded (very very) basic controller impls for each page; just writing to the response stream the name of the page